composer.json

```diff
@@ -12,10 +12,10 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "fsi/admin-bundle" : "~1.1@dev",
-        "symfony/framework-bundle": "~2.3",
```

Comment:
you just forced everybody who wants to use security bundle with admin bundle to use symfony version greater than 2.8. Which means projects created on 2.3 will can't be updated now - its huge BC Breake for them.